### PR TITLE
added welcome message

### PR DESCRIPTION
**Pull request by modernNeo**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -178,6 +178,33 @@ def createLogFile(formatter,logger):
...
+        output+="\t#sfu-discussions, for all SFU related discussion.\n"
+        output+="\t#projects_and_dev, for non-academic tech/dev/project discussion.\n"
+        output+="\t#bot_commands_and_misc, for command testing to reduce spam on other channels.\n"
+        output+="\n"
+        output+="\n"
+        output+="\tWe also have a smattering of course specific Academic channels.\n"
+        output+="\tYou can give yourself a class role by running <.iam cmpt320> or create a new class by <.newclass cmpt316>\n"
+        output+="\tPlease keep Academic Honesty in mind when discussing course material here.\n"
+
+        eObj = imported_embed(title="Welcome to the SFU CSSS's "+str(os.environ['BRANCH'])+" Discord Channel", author=settings.BOT_NAME, avatar=settings.BOT_AVATAR, description=output)
```
Comment (TitanVJ) [inline main.py]: Server not Channel

Comment (TitanVJ) [reply main.py]: also theres only one CSSS server, so i dont see why the need to specify branch

Comment (modernNeo) [reply main.py]: @TitanVJ I was debugging

**Comment by modernNeo** [comment]
@TitanVJ so here's where it stands, when I launch that code into my personal server, it works fine. but when I launch it into the test server, I get 2 dms. One is a message without the embed and one is a message with the embed. how important is this embed thing to you? @TitanVJ

**Comment by TitanVJ** [comment]
@modernNeo when i test locally i get one message. so it might be something with the test server. also not that important to me about the embed

**Comment by modernNeo** [comment]
i am closing this PR since the test server gets duplicate messages even when embeds arent used

**Comment by wnfldchen** [comment]
I know exactly what happened and it has to do with the test server not cleaning up old branches after a PR is opened and after they are deleted as well. That is a priority fix on my part.

**Comment by modernNeo** [comment]
Oh, in that case I will re-open it and wait for you to resolve that

**Comment by wnfldchen** [comment]
So this *should* work as intended when issue #114 is done

**Comment by modernNeo** [comment]
@TitanVJ and @wnfldchen this is OK to be approved now

**Pull request merged**